torch/autocast_mode.py

```diff
@@ -0,0 +1,163 @@
+import torch
+import functools
+
+class autocast(object):
+    r"""
+    Instances of :class:`autocast` serve as context managers or decorators that

```

Comment:
Please re-copy the [original docstring](https://github.com/pytorch/pytorch/pull/61002/files#diff-dac4bd53ced015c8810b3b02fc5c2ec6c2b0658c5090b4fbbd09c96bd45087d1L16) to make sure the formatting and newlines match exactly, as the newlines _will_ affect the rendered paragraphs.

Did you make any changes to the actual text of the original docstring?

Reply:
Rendered docs can be previewed at https://docs-preview.pytorch.org/61002/ (this link is in Dr.CI's comment)